nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -6,6 +6,21 @@
 
 from nautobot.core.utils.lookup import get_model_for_view_name
 
+# List of view names that are excluded for various error responses
+EXCLUDED_VIEW_NAMES = [
+    # "You do not have permission to perform this action." even when logged in as superuser
+    "apps-api:apps-list",
+    "plugins-api:plugins-list",  # same as above
```

Comment:
Is the superuser `is_staff=True`? This view has `permission_classes = [permissions.IsAdminUser]`